Packages/vcs/Lib/VCS_validation_functions.py

```diff
@@ -877,6 +883,10 @@ def checkLineType(self, name, value):
         hvalue = 'long-dash'
     elif (queries.isline(value) == 1):
         hvalue = value.name
+    elif value in vcs.elements["line"]:
+        self.linecolor = vcs.elements["line"][value].color[0]
```

Comment:
If `vcs.elements["line"][value]` returns `None`, these lines would cause a crash.

If you think that will never happen, please ignore.